.gitignore

```diff
@@ -0,0 +1,3 @@
+/.scannerwork
+/target/
```

Comment:
@lindgrenj6 What ends up in this `target` directory?  Should it be more descriptive name?

Reply:
Oh, that's just where maven puts its build files. Makes sense to ignore it, the `.gitignore` wasn't there before so I added it. I can remove it and add it in another PR if you'd like.

Reply:
Ok, not a problem.  You can leave it.